needls/server.py

```diff
@@ -512,22 +515,32 @@ def update_settings(ls, *args):
         )
         return
 
-    # check if config for confPath exists
-    if len(args[0]) >= 4:
-        # check if path is relative path
-        if not os.path.isabs(args[0][3]):
-            conf_py_path = os.path.join(os.getcwd(), args[0][3])
-            ls.show_message_log(
-                f"Configuration file: relative path is given -> {args[0][3]}, need to caluculate to absolute path -> {conf_py_path}"
-            )
+    # using conf.py file under docsRoot by default
+    conf_py_path = os.path.join(docs_root, "conf.py")
+
+    if not os.path.exists(conf_py_path):
```

Comment:
I think the order is wrong.

You should first trust confPath, and if it is not set use "docs/conf.py".

There can be multiple sphinx-docs in a repo and always one with a "docs/conf.py".
But if you set confPath to ``example/my_docs/folder/conf,py``, ``docs/conf.py`` is still taken.